functions/binarySearch.js

```diff
@@ -0,0 +1,19 @@
+export default function binarySearch(sortedArray, number) {
+
+  let L = 0;
```

Comment:
These variable names are not informative at *all*. Please use better variable names